skycoin/skywire
# Fix/vpn server offline error

Did you run `make format && make check`?
yes

Fixes #1181

 Changes:	
- Added new packet `ErrorPacket`
- Added new error in vpn-client whitelist

How to test this PR:
1. Start visor A
2. Start visor B
3. From visor A use vpn-client to connect to visor B's pk
4. Check `no listener on port 44, vpn-server offline` error on UI
